src/utils/platform.js

```diff
@@ -0,0 +1,19 @@
+import { Platform } from 'react-native';
+
+export const IOS = 'ios';
+export const ANDROID = 'android';
+export const BOTH_PLATFORMS = 'both';
+
+export const IS_IOS = Platform.OS === IOS;
+export const IS_ANDROID = Platform.OS === ANDROID;
+
+export const platformEnabler = (platform = BOTH_PLATFORMS) => {
+  if (
+    platform === BOTH_PLATFORMS ||
+    (platform === IOS && IS_IOS) ||
+    (platform === ANDROID && IS_ANDROID)
+  ) {
+    return true;
+  }
+  return false;
+};
```

Comment:
You can just return the condition, like 
`export const platformEnabler = (platform = BOTH_PLATFORMS) => (condition);`

Reply:
This code was a copy from the accessibility repo and I think due to the long condition having it this way improves readability

Reply:
🙈 don't know why that if what left like that. I'm ok with returning the condition directly